src/App/App.js

```diff
@@ -1,18 +1,32 @@
 import React from 'react';
+import { BrowserRouter, Route } from "react-router-dom";
 
 import Header from '../Components/Header/header';
 import NavBar from '../Components/Navbar/nav';
 import Profile from '../Components/Profile/Profile';
+import Dialogs from '../Components/Dialogs/dialogs';
+import News from '../Components/News/news';
+import Music from '../Components/Music/music';
+import Settings from '../Components/Settings/settings';
 
 import './App.css';
 
+
 function App() {
   return (
-    <div className="wrapper">
-      <Header />
-      <NavBar />
-      <Profile />
-    </div>
+    <BrowserRouter>
+      <div className="wrapper">
+        <Header />
+        <NavBar />
+        <div className='app-wrapper-contnent'>
+          <Route path="/profile" component={Profile} />
+          <Route path="/dialogs" component={Dialogs} />
+          <Route path="/news" component={News} />
+          <Route path="/music" component={Music} />
+          <Route path="/settings" component={Settings} />
+        </div>
+      </div></BrowserRouter>
```

Comment:
I think there's a problem with indentation for BrowserRouter.

Reply:
Okey,I changed this problem.
![image](https://user-images.githubusercontent.com/46630057/74612330-8114ab80-5115-11ea-8f1e-3ee4616faeca.png)